pkg/build/builder/docker.go

```diff
@@ -66,7 +68,14 @@ func (d *DockerBuilder) Build() error {
 	if err != nil {
 		return err
 	}
-	sourceInfo, err := fetchSource(d.dockerClient, buildDir, d.build, initialURLCheckTimeout, os.Stdin, d.gitClient)
+	startTime := unversioned.Now()
+	sourceInfo, stages, err := fetchSource(d.dockerClient, buildDir, d.build, initialURLCheckTimeout, os.Stdin, d.gitClient)
+	stopTime := unversioned.Now()
+	var subStepDuration int64
```

Comment:
you don't need to declare this, you can just use := below (assuming this value even sticks around, but per my other comments i hope it doesn't)